src/server/src/models/Post.ts

```diff
@@ -109,6 +113,10 @@ Post.init(
       allowNull: false,
       defaultValue: false,
     },
+    comments: {
```

Comment:
This field is not required

Reply:
I added it because I saw the following, but I have removed it now
![Screenshot_20220328_142944](https://user-images.githubusercontent.com/43073294/160481708-4a44ce63-5075-4a53-8c5d-66bdbbe46985.png)